apps/next/src/pages/AccountManagement/components/ImportPrivateKey/Page.tsx

```diff
@@ -0,0 +1,6 @@
+import { FC } from 'react';
+import { ImportPrivateKeyForm } from './components/ImportPrivateKeyForm';
+
+export const ImportPrivateKey: FC = () => {
+  return <ImportPrivateKeyForm />;
+};
```

Comment:
Since the  `ImportPrivateKey` component does not contribute anything of value, 
please consider to just re-export `ImportPrivateKeyForm` component as `ImportPrivateKey`.

```suggestion
export { ImportPrivateKeyForm  as ImportPrivateKey } from './components/ImportPrivateKeyForm';
```